README.md

```diff
@@ -41,7 +41,11 @@ yarn test
 
 ## Contributions
 
-See [CONTRIBUTING.md](https://github.com/googleads/ad-speed-insights/blob/master/lighthouse-plugin-ad-speed-insights/CONTRIBUTING.md)
+See [CONTRIBUTING.md](https://github.com/googleads/pub-ads-lighthouse-plugin/blob/master/lighthouse-plugin-ad-speed-insights/CONTRIBUTING.md)
+
+## Web App
+
+We currently have a web app version of Publisher Ads Audits for Lighthouse. It is currently in preview, if you'd like to request access you can do so [here](https://forms.gle/1tYvRjrkwPXYUpLW8).
```

Comment:
More concisely:
A web app version of Publisher Ads Audits for Lighthouse is currently in preview, you can request access [here](https://forms.gle/1tYvRjrkwPXYUpLW8).